_episodes/01-why-bother-performance.md

```diff
@@ -141,14 +136,14 @@ The `--time` variable used in the exercise is the amount of CPU hours requested,
 > 3. Optimising the code
 > 4. Having a computer with higher flops
 > 
-> > ## Solution
-> > 1. Yes, the more cores and nodes you have, the more work can be distributed across those cores.
-> > 2. No, increasing simulation walltime and CPUh only increases the duration the code will run for. It does not improve the code performance.
-> > 3. Yes, making the code more efficient by cutting out unnecessary loops and bottlenecks can greatly increase performance.
-> > 4. Yes, the faster the computer, the faster the code can run, however, this may not always be possible logistically.
-> >
-> > As you can see, there are a lot of right answers, however some methods work better than others, and it can entirely depend on the problem you are trying to solve.
-> >
+>> ## Solution
+>> 1. Yes, the more cores and nodes you have, the more work can be distributed across those cores.
+>> 2. No, increasing simulation walltime and CPUh only increases the duration the code will run for. It does not improve the code performance.
+>> 3. Yes, making the code more efficient by cutting out unnecessary loops and bottlenecks can greatly increase performance.
```

Comment:
I wouldn't give an example here, performance optimisation is a minefield, say `making the code more efficient by identifying and improving bottlenecks can greatly increase performance`